### ISSUE TITLE
Remove the trend chart for non numeric types

### ISSUE THREAD

**Issue by nbiton** [nbiton/IoThingies#16]
Well, it doesn't makes much sense, does it?

**Comment by Nexus7** [comment]
No, it doesn't... I'm just thinking... when you set up a new datapoint should you be prompted to 'categorise' the types? My initial thought was choose between : 

i) boolean (eg, a light switch monitor)
ii) number (eg, the vast majority of sensor readings such as temperature)
iii) text (eg, your microcontroller may construct a text string and you might just want to see that)
iv) image URL (eg, link to a webcam monitor)
